.storybook/preview.tsx

```diff
@@ -38,6 +38,13 @@ addDecorator((StoryFn) => (
 export const parameters = {
   // https://storybook.js.org/docs/react/essentials/actions#automatically-matching-args
   actions: { argTypesRegex: "^on.*" },
+  backgrounds: {
+    default: "light",
+    values: [
```

Comment:
This is great.  Let’s expand upon this a bit.  Use the following for this array:
```
{ name: “Light mode page background", value: “#FFFFFF" },
{ name: “Light mode default background", value: "F5F5F5" },
{ name: “Dark mode page background", value: “#121212" },
{ name: “Dark mode default background", value: “#191919" },
```

Reply:
@oliviawongnyc Ugh.  I had a typo in my suggested code.  I forgot the `#` for the light mode default background.  And because of that, the color is not being applied when selected.  That, obviously, needs to be fixed.

The dark mode colors are working fine.  Yes, they are close, but that is intentional.

Reply:
Oops, I should have caught it too. Fixed now.